BUILD.bazel

```diff
@@ -1,88 +1,269 @@
+
+        # Expression
+        "//plugins/expression/ios:PlayerUIExpressionPlugin_Sources": "plugins/expression/ios/",
+        "//plugins/expression/core:core_native_bundle": "plugins/expression/ios/Resources/",
+
+        # External Action
+        "//plugins/external-action/ios:PlayerUIExternalActionPlugin_Sources": "plugins/external-action/ios/",
+        "//plugins/external-action/swiftui:PlayerUIExternalActionViewModifierPlugin_Sources": "plugins/external-action/swiftui/",
+        "//plugins/external-action/core:core_native_bundle": "plugins/external-action/ios/Resources/",
+
+        # Metrics
+        "//plugins/metrics/swiftui:PlayerUIMetricsPlugin_Sources": "plugins/metrics/swiftui/",
+        "//plugins/metrics/core:core_native_bundle": "plugins/metrics/swiftui/Resources/",
+
+        # Pending Transaction
+        "//plugins/pending-transaction/swiftui:PlayerUISwiftUIPendingTransactionPlugin_Sources": "plugins/pending-transaction/swiftui/",
+
+        # PubSub
+        "//plugins/pubsub/ios:PlayerUIPubSubPlugin_Sources": "plugins/pubsub/ios/",
+        "//plugins/pubsub/core:core_native_bundle": "plugins/pubsub/ios/Resources/",
+
+        # Reference Assets
+        "//plugins/reference-assets/swiftui:PlayerUIReferenceAssets_Sources": "plugins/reference-assets/swiftui/",
+        "//plugins/reference-assets/swiftui/Resources:PlayerUIReferenceAssets_Resources": "plugins/reference-assets/swiftui/Resources/",
+        "//plugins/reference-assets/core:core_native_bundle": "plugins/reference-assets/swiftui/Resources/js/",
+
+        # Stage Revert Data
+        "//plugins/stage-revert-data/ios:PlayerUIStageRevertDataPlugin_Sources": "plugins/stage-revert-data/ios/",
+        "//plugins/stage-revert-data/core:core_native_bundle": "plugins/stage-revert-data/ios/Resources/",
+
+        # Transition
+        "//plugins/transition/swiftui:PlayerUITransitionPlugin_Sources": "plugins/transition/swiftui",
+
+        # Types Provider
+        "//plugins/types-provider/ios:PlayerUITypesProviderPlugin_Sources": "plugins/types-provider/ios/",
+        "//plugins/types-provider/core:core_native_bundle": "plugins/types-provider/ios/Resources/",
     },
     podspec = ":PlayerUI_Podspec",
 )
 
-# Push podspec to specs repo
-# tag must exist in github first
-pod_push(
-    name = "PlayerUI_Pod_Push",
-    executable = "bundle exec pod",
-    globalFlags = [],
-    podspec = ":PlayerUI_Podspec",
-    pushFlags = [
-        # skip tests because it never runs them right
-        # and they're run as part of the build pipeline anyway
-        "--skip-tests",
+# # Push podspec to specs repo
+# # tag must exist in github first
+# pod_push(
+#     name = "PlayerUI_Pod_Push",
+#     executable = "bundle exec pod",
+#     globalFlags = [],
+#     podspec = ":PlayerUI_Podspec",
+#     pushFlags = [
+#         # skip tests because it never runs them right
+#         # and they're run as part of the build pipeline anyway
+#         "--skip-tests",
+#     ],
+# )
```

Comment:
@hborawski do we still need this?

Reply:
no, not needed, was hoping to swap it for the new publishing rule but havent been able to test yet